experiments/modern/src/App.js

```diff
@@ -144,6 +145,11 @@ function Button({ id, image, action, x, y, downImage, tooltip, children }) {
           setDown(false);
         });
       }}
+      onClick={e => {
+        if (hooks.includes("onLeftClick")) {
```

Comment:
Why do we need this check? We should be able to trigger the event even if nobody is listening.

Reply:
My intention is that we would not be binding event handlers unless the Maki object actually has that hook attached.

So we don't *need* the check, and obviously the check is pretty useless if we already bind the click handler anyway.

It might not be "necessary", but we will be firing a lot of ignored events for `onMouseMove` and the many of hooks that will fire pretty continuously otherwise. Since it seemed totally doable to actually "know" which handlers we need to fire for based on existing hooks, I tried to wire that up.